receiver/prometheusreceiver/internal/transaction_test.go

```diff
@@ -197,14 +270,42 @@ func nopObsRecv() *obsreport.Receiver {
 	})
 }
 
+func createTestExemplar(value float64, ts int64, ls labels.Labels) exemplar.Exemplar {
+	return exemplar.Exemplar{
+		Value:  value,
+		Ts:     ts,
+		Labels: ls,
+	}
+}
+
+func createTestExemplarWithoutTimestamp(value float64, ls labels.Labels) exemplar.Exemplar {
+	return exemplar.Exemplar{
+		Value:  value,
+		Labels: ls,
+	}
+}
+
+func createTestExemplarWithoutValue(ts int64, ls labels.Labels) exemplar.Exemplar {
+	return exemplar.Exemplar{
+		Ts:     ts,
+		Labels: ls,
+	}
+}
```

Comment:
These should go close to `createDataPoint` and other helpers.

Reply:
Removed all three methods during modification